tests/models/biogpt/test_modeling_biogpt.py

```diff
@@ -273,7 +273,10 @@ def setUp(self):
         self.config_tester = ConfigTester(self, config_class=BioGptConfig, hidden_size=37)
 
     def test_config(self):
-        self.config_tester.run_common_tests()
+        def check_fn(attribute):
+            return attribute == "layer_norm_eps"
```

Comment:
This is the way to allow some cases where an attribute usage is not found but we know clearly the reason and cannot change anything